go.mod

```diff
@@ -1,32 +1,32 @@
 module github.com/openshift/cluster-cloud-controller-manager-operator
 
-go 1.22.5
+go 1.23.0
 
 require (
 	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d
 	github.com/go-logr/logr v1.4.2
 	github.com/golangci/golangci-lint v1.62.2
-	github.com/onsi/ginkgo/v2 v2.20.2
-	github.com/onsi/gomega v1.34.2
-	github.com/openshift/api v0.0.0-20241001152557-e415140e5d5f
-	github.com/openshift/client-go v0.0.0-20241001162912-da6d55e4611f
+	github.com/onsi/ginkgo/v2 v2.21.0
+	github.com/onsi/gomega v1.35.1
+	github.com/openshift/api v0.0.0-20250128061932-d2791f98ff08
+	github.com/openshift/client-go v0.0.0-20250125113824-8e1f0b8fa9a7
 	github.com/openshift/cluster-api-actuator-pkg/testutils v0.0.0-20240607201500-81075cf8e11a
-	github.com/openshift/library-go v0.0.0-20241104101214-d62fbd9f01cf
+	github.com/openshift/library-go v0.0.0-20250127111945-0f76e23726cd
 	github.com/spf13/cobra v1.8.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.10.0
 	gopkg.in/gcfg.v1 v1.2.3
 	gopkg.in/ini.v1 v1.67.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.31.2
-	k8s.io/apiextensions-apiserver v0.31.2
-	k8s.io/apimachinery v0.31.2
-	k8s.io/client-go v0.31.2
+	k8s.io/api v0.32.1
+	k8s.io/apiextensions-apiserver v0.32.1
+	k8s.io/apimachinery v0.32.1
+	k8s.io/client-go v0.32.1
 	k8s.io/cloud-provider-vsphere v1.31.1
-	k8s.io/component-base v0.31.2
-	k8s.io/controller-manager v0.31.2
+	k8s.io/component-base v0.32.1
+	k8s.io/controller-manager v0.32.1
 	k8s.io/klog/v2 v2.130.1
-	k8s.io/utils v0.0.0-20240921022957-49e7df575cb6
+	k8s.io/utils v0.0.0-20241104100929-3ea5e8cea738
 	sigs.k8s.io/cloud-provider-azure v1.30.3
 	sigs.k8s.io/cloud-provider-azure/pkg/azclient v0.0.25
 	sigs.k8s.io/controller-runtime v0.19.1
```

Comment:
Should we be updating controller-runtime and other packages like cloud-provider-azure while we are here?